conf/messages.en

```diff
@@ -71,9 +71,13 @@ error.submissionError.link = start your claim again
 
 # Session Expired
 # ----------------------------------------------------------
-session_expired.title = For your security, this service has been reset
-session_expired.heading = For your security, this service has been reset
-session_expired.guidance = The details you have given have been deleted because you did not continue the service for 15 minutes.
+pageNotFound.title = Page not found - Error - Register your charity’s details with HMRC - GOV.UK
```

Comment:
remove  **- Register your charity’s details with HMRC - GOV.UK(W)**.. this will create duplicate in title